content/library/advanced-features/connection-to-data.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Connecting to data
```

Comment:
is it confusing to name the file "connection-to-data" when the title / slug is connectING to data?

Reply:
Good catch! Thankfully, the filename doesn't factor in to the structure of the docs site in any meaningful way (we recursively traverse all the files and parse the YAML front matter that the top of files. We try to mirror filenames and directory structure similar to the layout of the docs for an easier authoring experience. But the relevant bit is the front matter at the top of each file, specifying the title and slug. 

That said, the incorrect filename was a typo. Thanks for calling it out! Fixed on b081add.